__tests__/sources.ts

```diff
@@ -279,6 +279,31 @@ query Source($id: ID!) {
   });
 });
 
+describe('query sourceHandleTaken', () => {
```

Comment:
Please add test case to validate that the an error is thrown if not authenticated